docs/cli/configuration.md

```diff
@@ -185,6 +185,45 @@ In addition to a project settings file, a project's `.gemini` directory can cont
     "hideTips": true
     ```
 
+- **`model`** (string)
+  - **Description:** Specifies the Gemini model to use.
+  - **Default:** Empty
+  - **Example:**
+    ```json
+    "model": "gemini-2.5-flash"
+    ```
+
+- **`auth`** (object)
+  - **Description:** Configures the authentication settings for Gemini and Vertex AI.
+  - **Default:** Empty
+  - **Properties:**
+    - **`gemini`** (object): Gemini-specific authentication settings.
+      - **`apiKey`** (string): Your API key for the Gemini API.
+    - **`vertex`** (object): Vertex AI-specific authentication settings.
+      - **`apiKey`** (string): Google Cloud API key. Required for using Vertex AI in express mode.
+      - **`project`** (string): Google Cloud Project ID. Required for using Vertex AI in non-express mode.
+      - **`location`** (string): Google Cloud Project Location (e.g., us-central1). Required for using Vertex AI in non-express mode.
+      - **`labels`** (object):  Optional. The [labels](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.endpoints/generateContent#body.request_body.FIELDS.labels) with user-defined metadata for the Vertex AI API requests.
+  - **Example:**
+    ```jsonc
+    "selectedAuthType": "oauth", // oauth, gemini, or vertex
+    "auth": {
+      // for 'gemini'
+      "gemini": {
+        "apiKey": "YOUR_GEMINI_API_KEY"
```

Comment:
is the idea that this and other API keys be would be read from `.env`? or is it a literal string in settings.json?